.github/CONTRIBUTING.md

```diff
@@ -24,6 +24,9 @@ Generally speaking, you should fork this repository, make changes in your own fo
 
 All pipeline library coding _must_ come with automated unit tests.
 
+Beside that we have an integration test suite. This test are not triggered during normal pull request builds. These tests are mandatory before a change can be merged. It is the duty of the team member of the SAP/jenkins-library project to execute these tests.
```

Comment:
These tests are not triggered .., and they are mandatory (maybe to not repeat these tests again)

Reply:
And I think that it would be better "a team member", or, "the team member that reviews the PR", or something like this.
But @SarahNoack knows this better.